src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -619,18 +619,18 @@ class JsonPrinter : public Printer {
     {
         if (processRecordTypes.d_message) {
             closeBraceIfOpen();
-            d_ostream << "  \"TotalMessages\": \"" << foundMessagesCount
-                      << "\"";
+            d_ostream << "  \"TotalMessages\": " << foundMessagesCount
+              << "\"";
         }
         if (processRecordTypes.d_queueOp) {
             closeBraceIfOpen();
-            d_ostream << "  \"QueueOpRecords\": \"" << foundQueueOpCount
-                      << "\"";
+            d_ostream << "  \"QueueOpRecords\": " << foundQueueOpCount
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.